symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/MessageBuilder.java

```diff
@@ -0,0 +1,139 @@
+package com.symphony.bdk.core.service.message.model;
+
+import static java.util.Collections.emptyMap;
+
+import com.symphony.bdk.core.service.message.MessageService;
+import com.symphony.bdk.core.service.message.exception.MessageCreationException;
+import com.symphony.bdk.gen.api.model.V4Message;
+import com.symphony.bdk.gen.api.model.V4Stream;
+import com.symphony.bdk.template.api.Template;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+import java.io.InputStream;
+
+/**
+ * Builder for {@link Message}.
+ * Allows configuring the field of the messages before constructing them with build()
+ */
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class MessageBuilder {
+
+  private static final ObjectMapper MAPPER = new JsonMapper();
+
+  private final MessageService messageService;
+  private String version = "2.0";
+  private String content;
+  private String data;
+  private Attachment attachment;
+
+  public MessageBuilder(MessageService messageService) {
+    this.messageService = messageService;
+  }
+
+  /**
+   * Add messageML content to the message.
+   *
+   * @param   message    messageML.
+   * @return  this builder with the content configured.
+   */
+  public MessageBuilder messageML(String message) {
```

Comment:
why not just `content`? As you mention in the Javadoc that it is expecting MessageML